test/presenters/taxon_presenter_test.rb

```diff
@@ -216,6 +216,14 @@
   end
 
   describe 'guidance_and_regulation_section' do
+    it 'checks whether guidance section should be shown' do
+      taxon = mock
```

Comment:
I'd use a `before` block for this setup and a `let` block for `TaxonPresenter.new(taxon)`.